PointOfDispensingApp/src/components/PatientLookupPage.vue

```diff
@@ -156,6 +185,18 @@
         //Advance to the Check In page
         this.$router.push("CheckIn")
       },
+      formatDate (date) {
+        if (!date) return null
+
+        const [year, month, day] = date.split('-')
+        return `${month}/${day}/${year}`
+      },
+      parseDate (date) {
+        if (!date) return null
+
+        const [month, day, year] = date.split('/')
```

Comment:
This doesnt work as expected when the text field input uses other separators. Maybe use v-mask on the text field?